MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -0,0 +1,277 @@
+#import "MSPush.h"
+#import "MSPushLog.h"
+#import "MSPushPrivate.h"
+#import "MSPushInternal.h"
+#import "MSDeviceTracker.h"
+#import "MSMobileCenterInternal.h"
+
+#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+
+#import <UserNotifications/UserNotifications.h>
+
+#endif
+
+/**
+ * Service storage key name.
+ */
+static NSString *const kMSServiceName = @"Push";
+
+/**
+ * Key for storing push token
+ */
+static NSString *const kMSPushServiceStorageKey = @"kmspushservicepushstoringkey";
+
+/**
+ * Singleton
+ */
+static MSPush *sharedInstance = nil;
+static dispatch_once_t onceToken;
+
+@implementation MSPush
+
+@synthesize deviceTokenHasBeenSent;
+@synthesize isRequestInProgress;
+
+#pragma mark - Service initialization
+
```

Comment:
Remove all empty lines in the methods in this file. Line breaking doesn't require unless there is a comment.